lib/node_modules/@stdlib/blas/ext/base/dapx/README.md

```diff
@@ -117,7 +113,7 @@ dapx.ndarray( 3, 5.0, x, 1, x.length-3 );
 
 ## Notes
 
--   If `N <= 0`, both functions return `x` unchanged.
+-   If `N <= 0`, both functions return the strided array unchanged.
 
 </section>
 
```

Comment:
@naveen1m Would you mind updating the example below to use `discrete-uniform`? The values should be between `-100` and `100`. Thanks!